system-addon/lib/TelemetryFeed.jsm

```diff
@@ -256,8 +353,13 @@ this.TelemetryFeed = class TelemetryFeed {
       case at.SAVE_SESSION_PERF_DATA:
         this.saveSessionPerfData(au.getPortIdOfSender(action), action.data);
         break;
+      case at.SYSTEM_TICK:
+        if (Date.now() - this._impressionStatsLastReset >= IMPRESSION_STATS_RESET_TIME) {
+          this.resetImpressionStats();
+        }
```

Comment:
Is it intentional that we reset after the first system tick (5 mins and then every 60 mins) or should we initialize ```this._impressionStatsLastReset = Date.now();``` instead of ```this._impressionStatsLastReset = 0;```